### DISCUSSION TITLE
AOSP 11.0 v311

### DISCUSSION THREAD

**Discussion by turuncu2011** [phhusson/treble_experimentations#1962] [Q&A]
AOSP 11.0 v311, status bar, nav bar and sound panel are still missing. Why don't you explain a solution for this?

**Comment by turuncu2011** [comment]
APPEAL YOU DON'T TAKE US SO SERIOUSLY.

**Reply by phhusson** [reply]
I don't explain a solution for it, because this is an issue, and this isn't the place to fix issues.
You see, there is a big button called "issues" on your webpage.

Click there, instead of going into the forum part

And then, you're not asking how to fix this, you're making the manager make-stence, asking for explanations on how I work.
You have to actually pay me a salary to have any right to do this.

**Reply by turuncu2011** [reply]
Earlier versions did not have this problem. It started in new versions, is there something wrong with me?  I just wanted to ask, thank you for your hard work.
